files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -203,6 +203,22 @@ toggle-gnome-jit-js:
     	echo "JIT JavaScript for Gnome and WebkitGTK has been disabled."
     fi
 
+# Toggle support for using GNOME user extensions
+toggle-gnome-extensions:
+    #!/usr/bin/bash
+    GSETTING="$(gsettings get org.gnome.shell allow-extension-installation)"
+    CURRENT_STATE="Disabled"
+    if [[ "${GSETTING}" == "true" ]]; then
+      CURRENT_STATE="Enabled"
+    fi
+    if [[ "${CURRENT_STATE}" == "Disabled" ]]; then
```

Comment:
This can be simplified and CURRENT_STATE can be removed entirely

you can just do

if GSETTING == "true" then

else